Changes

```diff
@@ -1,3 +1,6 @@
+2.16  2023-09-19
+  - Adapt to deprecation of spurt in upstream Mojolicious
+    Contributor: Oliver Kurz
```

Comment:
The maintainer writes the Changes file, and 2.15 is not even released yet.

Reply:
done